frontend/src/components/ConfigurationP2.tsx

```diff
@@ -126,6 +141,36 @@ export function Controller(){
   };
   
   const handleSubmit = () => {
+     // Here we will make the requests to the backend to store the config info and to begin the integration and redirect to second config screen
+     const postConfigData = async () => {
+      return await fetch(API_CONFIGURATION_URL, {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({ 
+          ConnectToHubspot: false,
```

Comment:
Should this be false? Wouldn't this overwrite the config we saved on the first config page?

Reply:
So I just put it as false for now! We'll need to keep track of the data on both pages for if a user wants to go back or forward in the flow and I don't think we have that set up yet as right now the config data is confined to it's own page.